### ISSUE TITLE
Add block hash and height to api result CoinbaseTxObject and SpendTxObject

### ISSUE THREAD

**Issue by dadaphl** [aeternity/protocol#15]
Scenarios
- figure out the confirmations of a transaction
- traverse to the related block to get the timestamp

Affect api endpoints
```
GET /block/hash/{hash}
GET /block/tx/hash/{hash}/{tx_index}
GET /block/txs/list/height
GET /block/txs/list/hash
```

For consistency could also affect all endpoints that include transaction objects, even if it's clear in which block the transaction was mined, given by the request itself.
```
GET /block/height/{height}
GET /block/genesis
GET /block/latest
GET /block/tx/height/{height}/{tx_index}
GET /block/tx/latest/{tx_index}
```

**Comment by dadaphl** [comment]
Height is ambiguous and potentially dangerous to add because of forking. Not sure if it should be included.

**Comment by velzevur** [comment]
Implemented in aeternity/epoch#644
